support-models/src/main/scala/com/gu/support/catalog/Product.scala

```diff
@@ -39,17 +40,23 @@ case object DigitalPack extends Product {
       PROD -> List(
         ProductRatePlan("2c92a0fb4edd70c8014edeaa4eae220a", Monthly, NoFulfilmentOptions, NoProductOptions, "Digital Subscription Monthly"),
         ProductRatePlan("2c92a0fb4edd70c8014edeaa4e972204", Annual, NoFulfilmentOptions, NoProductOptions, "Digital Subscription Annual"),
-        ProductRatePlan("2c92a00d71c96bac0171df3a5622740f", Monthly, NoFulfilmentOptions, Corporate, "Digital Subscription Redemption Code"),
+        ProductRatePlan("2c92a00d71c96bac0171df3a5622740f", Monthly, NoFulfilmentOptions, NoProductOptions, "Digital Subscription Redemption Code",
+          readerType = Corporate
+        ),
       ),
       UAT -> List(
         ProductRatePlan("2c92c0f94f2acf73014f2c908f671591", Monthly, NoFulfilmentOptions, NoProductOptions, "Digital Subscription Monthly"),
         ProductRatePlan("2c92c0f84f2ac59d014f2c94aea9199e", Annual, NoFulfilmentOptions, NoProductOptions, "Digital Subscription Annual"),
-        ProductRatePlan("2c92c0f971c65df50171dfabef87093d", Monthly, NoFulfilmentOptions, Corporate, "Digital Subscription Redemption Code"),
+        ProductRatePlan("2c92c0f971c65df50171dfabef87093d", Monthly, NoFulfilmentOptions, NoProductOptions, "Digital Subscription Redemption Code",
+          readerType = Corporate
+        ),
       ),
       SANDBOX -> List(
         ProductRatePlan("2c92c0f84bbfec8b014bc655f4852d9d", Monthly, NoFulfilmentOptions, NoProductOptions, "Digital Subscription Monthly"),
         ProductRatePlan("2c92c0f94bbffaaa014bc6a4212e205b", Annual, NoFulfilmentOptions, NoProductOptions, "Digital Subscription Annual"),
-        ProductRatePlan("2c92c0f971c65dfe0171c6c1f86e603c", Monthly, NoFulfilmentOptions, Corporate, "Digital Subscription Redemption Code"),
+        ProductRatePlan("2c92c0f971c65dfe0171c6c1f86e603c", Monthly, NoFulfilmentOptions, NoProductOptions, "Digital Subscription Redemption Code",
+          readerType = Corporate
```

Comment:
something funny about the formatting here, all the parameters are for the same `ProductRatePlan` object I think? but some of them are on one line and not others. (and mostly positional but one named)

if so I'd prefer to be consistent, either all on one line if readable, if not all on separate lines.
If it gets that we can't see the wood for the trees we can factor out the repetition a bit better.

Reply:
It's because the last one is the only one which supplies a `readerType` the others just accept the default of `Direct`. I'll rejig the formatting to make it a bit neater.